### ISSUE TITLE
Support for dapps (collection issue for dapps that don't work)

### ISSUE THREAD

**Issue by rmeissner** [gnosis/safe-browser-extension#81]
As a user I would like to be able to use ALL the dapps available with the Safe.

Currently some dapps are not working. These dapps will be collected here:

| Website     | Error           | 
| -----------|---------- |
| https://alchemy.daostack.io/ | Not debugged yet |
| https://manager.balance.io/metamask | Not debugged yet |

**Comment by germartinez** [comment]
https://alchemy.daostack.io/ calls `eth_accounts` syncronously

**Comment by rmeissner** [comment]
why are they doing this 😭

**Comment by germartinez** [comment]
Yes, I have this solved (for eth_accounts and eth_coinbase), but not pushed yet

**Comment by villanuevawill** [comment]
Bounties Network, Cryptokitties, 3box, Kauri and many more still use personal_sign. Since https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md is technically not in the completed stage, this implementation has been held off.  In order to have compatibility with the gnosis extension, we will need to include various refactors around this.

**Comment by jellegerbrandy** [comment]
dev on https://alchemy.daostack.io/ here. 
Just to let you know that we are using `personal_sign` as well, and so we have a problem integrating gnosis safe.
